internal/wireguard/config.go

```diff
@@ -68,6 +71,7 @@ func makeDeviceConfig(settings Settings) (config wgtypes.Config, err error) {
 					IP:   settings.Endpoint.Addr().AsSlice(),
 					Port: int(settings.Endpoint.Port()),
 				},
+				PersistentKeepaliveInterval: &persistenKeepAlive,
```

Comment:
Leave this to `nil` if the persistent keep alive is `0` perhaps? Or maybe it makes no difference if it's nil or 0.